src/Microsoft.Extensions.Logging.Abstractions/Microsoft.Extensions.Logging.Abstractions.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <Description>Logging abstractions for Microsoft.Extensions.Logging.
+Commonly used types:
+Microsoft.Extensions.Logging.ILogger
+Microsoft.Extensions.Logging.ILoggerFactory
+Microsoft.Extensions.Logging.ILogger&lt;TCategoryName&gt;
+Microsoft.Extensions.Logging.LogLevel
+Microsoft.Extensions.Logging.
```

Comment:
Huh, yeah, looks wrong, but it's been that way for almost a year. cc @Eilon who wrote this description?

![image](https://cloud.githubusercontent.com/assets/2696087/22517248/66c9c932-e85d-11e6-93a8-6208f46d2407.png)